docs/user-guide/api-mediation/configuration-extender-passtickets.md

```diff
@@ -61,106 +62,111 @@ To validate if a PassTicket is already defined, use the commands that correspond
     SET RESOURCE(PTK)
     LIST LIKE(IRRPTAUTH-)
     ```
-    * **`-`**  
+
+  - **`-`**  
     A wildcard symbol that lists all resources
 
-    * **`<applid>-`**  
+  - **`<applid>-`**  
     Lists everything related to specified applid in a resource (in this case, SAF), or specified in a profile (in this case, PTKTDATA)
 
     </details>
 
-* **Validating an existing PassTicket for Top Secret**
+- **Validating an existing PassTicket for Top Secret**
 
     <details>
 
     <summary>Click here for details about validating an existing PassTicket for Top Secret.</summary>
 
     In your ESM command line interface or other security environment, execute the following commands:
+
     ```
     TSS WHOHAS APPL(<applid>)
     TSS WHOHAS PTKTDATA(<applid>)
     TSS WHOHAS PTKTDATA(IRRPTAUTH.<applid>.)
     ```
 
-    * **`.`**  
+  - **`.`**  
     A wildcard symbol that lists all resources
 
-    * **`IRRPTAUTH.<applid>.`**  
+  - **`IRRPTAUTH.<applid>.`**  
     Returns everything about the specified applid for IRRPTAUTH
 
     </details>
 
-* **Validating an existing PassTicket for RACF**
+- **Validating an existing PassTicket for RACF**
 
     <details>
 
     <summary>Click here for details about validating an existing PassTicket for RACF.</summary>
 
     In your ESM command line interface or other security environment, execute the following commands:
 
-    ```
+    ```rac
     RLIST APPL * ALL 
     RLIST APPL <applid> ALL  
     RLIST PTKTDATA <applid> SSIGNON ALL
     RLIST PTKTDATA IRRPTAUTH.<applid>.* ALL 
     ```
+
     Ensure that you validate PKTDATA access for APPL.
 
-    * **`*`**  
+  - **`*`**  
     A wildcard symbol that resturns all resources
 
-    * **`RLIST PTKTDATA <applid> SSIGNON ALL`**  
+  - **`RLIST PTKTDATA <applid> SSIGNON ALL`**  
     Validates all applid for PTKDATA class
 
-    * **`RLIST PTKTDATA IRRPTAUTH.<applid>.* ALL`**  
+  - **`RLIST PTKTDATA IRRPTAUTH.<applid>.* ALL`**  
     Validates all applid permissions for PTKDATA class
 
     </details>
 
 :::
 
-Follow these steps to enable PassTicket Support specific to your ESM. Consult with your security administrator to perform the following procedures. 
+Follow these steps to enable PassTicket Support specific to your ESM. Consult with your security administrator to perform the following procedures.
 
 #### Enabling PassTickets with ACF2
 
 <details>
 
 <summary> Click here for details about configuring Zowe to use PassTickets using ACF2. </summary>
 
+1. In your ESM command line interface or other security environment, define the application session key by entering the following commands, if the session key is not already defined.
 
-1.	In your ESM command line interface or other security environment, define the application session key by entering the following commands, if the session key is not already defined. 
-
-```
+```acf2
 SET PROFILE(PTKTDATA) DIV(SSIGNON)
 INSERT <applid> SSKEY(<key-description>) MULT-USE
```

Comment:
MULT-USE is for no reply protection, it should be removed

Reply:
Thanks, updated